interfaces/builtin/broadcom_asic_control.go

```diff
@@ -38,12 +38,20 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 /dev/linux-user-bde rw,
 /dev/linux-kernel-bde rw,
 /dev/linux-bcm-knet rw,
+
+# These are broader than they needs to be, but until we query udev
+# for specific devices, use a broader glob
```

Comment:
I think it's related to hardware assignment. We have a bug on launchpad tracking on this. Not sure where are we with that. 
@jdstrand Could you share the updates and plan here?
Thanks